doc/pages/administrator/095-advanced-operations.md

```diff
@@ -4,6 +4,14 @@ This section provides guides to perform some operations that have to be perfomed
 could result in data loss or other type of irrecoverable damage. *Always be careful while performing
 these operations*
 
+The following topics will be treated in the following sections:
```

Comment:
Avoid repetition; "Advanced operations are described in the following sections:"